src/hooks/useLiveTasks.ts

```
@@ -0,0 +1,262 @@
+      for (const row of assignRows ?? []) {
+        const a = row as LiveTaskAssignment & {
+          profiles?: {
+            first_name: string | null;
+            last_name: string | null;
+            avatar_url: string | null;
+          };
+        };
+        const name = a.profiles
+          ? [a.profiles.first_name, a.profiles.last_name]
+              .filter(Boolean)
+              .join(" ")
```

ESLint

[error] Replace `⏎··············.filter(Boolean)⏎··············` with `.filter(Boolean)`

(prettier/prettier)